docs/CONTRIBUTING.md

```diff
@@ -154,19 +155,46 @@ yarn test
 yarn compile
 ```
 
-## :arrow_heading_up: Pull Request
+## :mag: Pull requests & Code reviews
+Once you are happy with your implementation it is time to open a pull request (PR). After doing so, your code will be 
+reviewed by the B01LER core team. Once the PR has two approvals, your code will be merged to develop and will be part 
+of a future release. 
 
-Once your contribution is ready, it is time to create a pull request. Once your contribution is ready, it is time to
-create a pull request. Please create pull requests for the `develop` branch and not for `main`. Please also make sure
-that pull requests are linked to the related issue by using a keyword and the issue number as explained
-[here](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
-or by using the `Development` field.
+In case we find things we would like you to change we will leave comments. We kindly ask you to update your pull request 
+in regards of the requested changes. Once all comments have been solved the pull request will be merged. We will try our 
+best to respond within a week to new pull requests. 
+
+Contributors are also invited to review other contributors pull requests, but in the end at least one member of the core 
+team needs to approve it, before it can be merged.
 
-## :mag: Code Review
 
-After you created a pull request, your code will be reviewed by the B01LER core team. Once the PR has two approvals,
-your code will be merged to develop and will be part of a future release. In case we find things we would like you to
-change we will leave comments. We kindly ask you to update your pull request in regards of the requested changes. Once
-all comments have been solved the pull request will be merged. We will try our best to respond within a week to new pull
-requests. Contributors are also invited to review other contributors pull requests, but in the end at least one member
-of the core team needs to approve it, before it can be merged.
+### Tips for working on a pull request
+- Please create pull requests for the `develop` branch and not for `main`.
+- Please also make sure that pull requests are linked to the related issue by using a keyword and the issue number as
```

Comment:
Just a suggestion "Make sure that a pull request is linked to the related issue ... " (because there will be only 1 PR per issue. Also removed 'Please also' part because each sentence is already a bullet point).